### ISSUE TITLE
Implement ARIA attribute mappings for HTML

### ISSUE THREAD

**Issue by kasperisager** [Siteimprove/alfa#195]
Currently, we lack an implementation of the ARIA attribute mappings for HTML as outlined by: https://w3c.github.io/html-aam/#html-attribute-state-and-property-mappings. Some work has been done by @Jym77 on https://github.com/Siteimprove/alfa/compare/aria-attributes but I'm unsure how far along and up-to-date that is.

**Comment by Jym77** [comment]
The aria-attributes branch is a bit of a mess. I merged in master. Some tests are failing. Need to investigate.
In any case, it was still at the PoC stage where I was trying to understand how Alfa does stuff. So it might be as simple to start it from the scratch again 🤷‍♂
